tensorboard/webapp/tbdev_upload/tbdev_upload_dialog_component.ng.html

```diff
@@ -39,3 +39,16 @@ <h3>Upload to TensorBoard.dev?</h3>
   Only certain plugins are currently supported. Uploaded TensorBoards are public
   and visible to everyone; do not upload sensitive data.
 </p>
+<div class="bottom-buttons">
+  <button mat-flat-button class="close-button" (click)="onClose()">
+    Close
+  </button>
+  <a
+    mat-flat-button
+    class="learn-more-button"
+    href="https://tensorboard.dev"
+    target="_blank"
```

Comment:
For <a> tags, please add `rel="noopener noreferrer"`, similar to other anchor tags.

Reply:
I added rel="noopener" but I would like to pass referrer information for tensorboard.dev analytics. Would that work?

Reply:
Ack, only `rel="noopener"` sounds fine for this case.

Reply:
Please hold on this pending internal discussion.

Reply:
After internal discussion we decided to include noreferrer.

Reply:
Thanks!